### PR TITLE
Drop mantle and kola-data from the SDK

### PR DESCRIPTION
**Pull request by pothos**
With the new mantle container image referenced by the scripts repo we
don't need the mantle copy in the SDK anymore.
Drop the mantle package and the unused kola-data package.

## How to use
pick for all channels

Goes with https://github.com/flatcar-linux/scripts/pull/280

## Testing done

[passed](http://192.168.42.7:8080/job/os/job/manifest/5465/cldsv/)

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)

### CODE REVIEW

**Comment by jepio** [comment]
Aren't bits of mantle used by various things in scripts?

**Comment by pothos** [comment]
If yes, they should use the container image, except for the original Jenkins pipeline that uses the binaries from Jenkins

**Comment by pothos** [comment]
looks like `jenkins/sdk.sh` and `jenkins/toolchains.sh` need to be touched to use the Jenkins binaries

**Comment by jepio** [comment]
We're dismantling the SDK?

**Comment by pothos** [comment]
I'll run a test build with https://github.com/flatcar-linux/scripts/pull/280

File: changelog/changes/2022-04-20-mantle-sdk.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- SDK: Dropped the mantle binaries (kola, ore, etc) from the SDK because they are now provided by the `ghcr.io/flatcar-linux/mantle` image ([PR#1827](https://github.com/flatcar-linux/coreos-overlay/pull/1827), [PR#275](https://github.com/flatcar-linux/scripts/pull/275))
```
Comment (tormath1) [inline changelog/changes/2022-04-20-mantle-sdk.md:1]: nit-pick:
```suggestion
- SDK: Dropped the mantle binaries (kola, ore, etc.) from the SDK. They are now provided by the `ghcr.io/flatcar-linux/mantle` image ([PR#1827](https://github.com/flatcar-linux/coreos-overlay/pull/1827), [PR#275](https://github.com/flatcar-linux/scripts/pull/275))
```

**Review by tormath1 — APPROVED** [review]
That looks good to me - we should maybe go through the documentation to search for Mantle SDK mentions and update them consequently.

**Pull request merged**